hoomd/md/module-md.cc

```diff
@@ -257,26 +257,26 @@ PYBIND11_MODULE(_md, m)
     export_PotentialPair<PotentialPairDPD>(m, "PotentialPairDPD");
     export_PotentialPair<PotentialPairMoliere>(m, "PotentialPairMoliere");
     export_PotentialPair<PotentialPairZBL>(m, "PotentialPairZBL");
-    export_PotentialTersoff<PotentialTripletTersoff>(m, "PotentialTersoff");
-    export_PotentialTersoff<PotentialTripletSquareDensity> (m, "PotentialSquareDensity");
-    export_PotentialTersoff<PotentialTripletRevCross> (m, "PotentialRevCross");
+    // export_PotentialTersoff<PotentialTripletTersoff>(m, "PotentialTersoff");
+    // export_PotentialTersoff<PotentialTripletSquareDensity> (m, "PotentialSquareDensity");
+    // export_PotentialTersoff<PotentialTripletRevCross> (m, "PotentialRevCross");
     export_PotentialPair<PotentialPairMie>(m, "PotentialPairMie");
     export_PotentialPair<PotentialPairReactionField>(m, "PotentialPairReactionField");
     export_PotentialPair<PotentialPairDLVO>(m, "PotentialPairDLVO");
     export_PotentialPair<PotentialPairFourier>(m, "PotentialPairFourier");
     export_tersoff_params(m);
     export_revcross_params(m);
     export_pair_params(m);
-    export_AnisoPotentialPair<AnisoPotentialPairGB>(m, "AnisoPotentialPairGB");
-    export_AnisoPotentialPair<AnisoPotentialPairDipole>(m, "AnisoPotentialPairDipole");
+    // export_AnisoPotentialPair<AnisoPotentialPairGB>(m, "AnisoPotentialPairGB");
```

Comment:
Why are so many of these commented out? This shouldn't be necessary any longer as all pair potentials have been updated.